season/beta_challenges/hashes.md

```diff
@@ -0,0 +1,112 @@
+# Hashing It Out
+## Movies
+Manually create a hash with the following keys/values:
+| Key | Value |
+|---|---|
+| 'Hitchcock' | ['Rear Window']|
+| 'Truffaut' | ['400 Blows'] |
+| 'Myers'  | ['Austin Powers', 'Shrek']|
+| 'fruit' | 'banana' |
+Programmatically do the following:
+```ruby 
+random = {:Hitchcock=>["Rear window"], :Truffaut=>["400 blows"], :Myers=>["Austin Powers", "Shrek"], :Fruit=>"Banana"}
+```
+1. `fruit` and `banana` aren't movies! Delete them from the hash.
+```ruby
+random.delete(:Fruit)
+```
+2. Add "The Birds" to the `Hitchcock` array.
+```ruby
+random[:Hitchcock].push("The birds")
+```
+3. Remove `Shrek`.
+```ruby
+random[:Myers].delete("Shrek")
+```
+4. Return all of the keys.
+```ruby
+random.keys    
+```
+5. Return all the values.
+```ruby
+random.values
+```
+## Quit Putin Me On
+```
+putins_brain = {
+  launch_code: "a5Mjp257GHMGH23e5qxE",
+  fav_hobby: "Riding ponies",
+  prideful: true
+}
+```
+Programmatically do the following:
+1. Return the string `"Riding ponies"`
+```ruby
+putins_brain[:fav_hobby]
+```
+2. Return the string `"a5Mjp257GHMGH23e5qxE"`
+```ruby
+putins_brain[:launch_code]
+```
+3. Add the key-value pair `obsessiveCrush: "Condoleezza Rice"`
+```ruby
+putins_brain.store(:obsessiveCrush, "Condoleezza Rice")
+```
+4. Add the key-value pair `torture_count: 931`
+```ruby
+putins_brain.store(:torture_count, 931)
+```
+## The Most Ridiculous Grocery List
+Create a hash with keys representing the following foods: avocados, steak, grass, whiskey, muffins, fish, and soap.
+
+
+Avocados are fresh, steak is meaty, grass is fiber-filled, whiskey is smooth, muffins are sweet, fish is mercury-tastic, and soap is disgusting. Here is some starter code:
+Starter code:
+```
+groceries = {
+  avocados: "fresh",
+  steak: "meaty",
+  grass: "fiber-filled",
+  whiskey: "smooth",
+  muffins: "sweet",
+  fish: "mercury-tastic",
+  soap: "disgusting",
+}
+```
+Programmatically do the following:
+1. Avocados are full of that good fat. Change the avocado value to be "fatty".
+```ruby
+groceries.store(:avocados, "fatty")
+```
```

Comment:
Interesting. I've never used the `.store` method. You could just do `groceries[:avocados] = "fatty"`. It's probably more performant?